hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleAccountTokens.java

```diff
@@ -165,6 +168,9 @@ public boolean includes(TokenID id) {
 	}
 
 	public void associateAll(Set<TokenID> ids) {
+		if (isImmutable()) {
+			throw new MutabilityException("Cannot associate any tokens to an immutable container");
+		}
```

Comment:
nit: This code snippet is repeated. We could extract it out into `throwOnImmutableContainter(errorMessage)`